src/ec2/model.ts

```diff
@@ -41,14 +41,17 @@ export class Ec2ConnectionManager {
         return new DefaultIamClient(this.regionCode)
     }
 
-    protected async getAttachedPolicies(instanceId: string): Promise<IAM.attachedPoliciesListType> {
+    protected async getAttachedPolicies(instanceId: string): Promise<IAM.AttachedPolicy[]> {
         const IamRole = await this.ec2Client.getAttachedIamRole(instanceId)
         if (!IamRole) {
```

Comment:
```suggestion
        if (!IamRole?.Arn) {
```

this would avoid the need for `IamRole!.Arn!` below